lib/parser/livesplit_core_parser.rb

```diff
@@ -25,25 +25,42 @@ def self.parse(run, fast: false)
       program = ExchangeFormat.to_s
     end
 
+    key_error_lambda = ->(_hash, key) { raise KeyError, "#{key} not found" }
+
     run = parse_result.unwrap
+    run_metadata = run.metadata
 
     run_object = {
       program: program,
       game: run.game_name,
+      game_icon: run.game_icon.presence,
       category: run.category_name,
-      srdc_id: run.metadata.run_id,
+      name: run.extended_name(false),
+      metadata: {
+        srdc_id: run_metadata.run_id.presence,
+        platform_name: run_metadata.platform_name.presence,
+        region_name: run_metadata.region_name.presence,
+        uses_emulator: run_metadata.uses_emulator,
+        variables: {}
+      },
```

Comment:
I'm confused by what goes in the `metadata` field vs. what doesn't. I see that you're inheriting how livesplit-core returns it, but still -- is there a rule of thumb for why e.g. `region_name` is a piece of metadata but `game_icon` isn't, or even `game` for that matter?

cc @CryZe @wooferzfg

Reply:
The metadata is mostly for the additional info we store for speedrun.com, kind of. The distinction is fairly arbitrary though.

Reply:
If you want me to add all run information to the metadata key I can, but up till now I was just putting what LSC classified as "metatdata" in it.